openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py

```diff
@@ -108,6 +157,8 @@ def student_view(self, context=None):
         """
         fragment = Fragment()
 
+        self.add_resource_urls(fragment)
+
         course = self.runtime.modulestore.get_course(self.course_key)
```

Comment:
@andy-armstrong Doesn't look like this is needed. I removed it and was able to interact with `DiscussionXBlock` instances as expected. See https://github.com/edx/edx-platform/pull/13483/commits/1abfac75b33cad44d24aceb65c95c67fb4f2bf34.